opdracht-1.js

```diff
@@ -2,12 +2,58 @@
 // Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen.
 // Log de uitkomst in de console.
 
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+    // Declareer een variabele die mbv inventory.map() over de array heen loopt
+    // Geef map() de parameter tvType
+    // Return de tvType.type
+    // Log de variabele
+
+const tvTypes = inventory.map((tvType) => {
+    return tvType.type;
+})
+
+console.log(tvTypes);
 
 // Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn.
 // Log de uitkomst in de console.
+    // Declareer een variabele
+    // Loop met een filter over de array
+    // Geef de filter de functie mee die vergelijkt of de originalStock gelijk is aan sold
+    // Indien dit waar is, moet het object toegevoegd worden aan de nieuwe array
+    // Log de nieuwe array
+
+const soldOutTvs = inventory.filter((soldOutTv) => {
+    return soldOutTv.sold === soldOutTv.originalStock;
+});
+
+console.log(soldOutTvs);
 
 // Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken.
 // Log de uitkomst in de console.
+    // Declareer een variabele AmbiLights
+    // Itereer met de filter(ambiLight) over de array heen
+    // Indien .option.amblight === true return dan de value
+    // Log de variabele
+
+const ambiLights = inventory.filter((ambiLight) => {
+    return ambiLight.options.ambiLight === true;
+})
```

Comment:
Omdat `ambiLight` van zichzelf al een boolean is hoef je er niet expliciet nog die check achter te zetten
```suggestion
const ambiLights = inventory.filter(ambiLight => ambiLight.options.ambiLight);
```